calico_containers/pycalico/netns.py

```diff
@@ -51,6 +51,20 @@ def setup_logging(logfile, level=logging.INFO):
     _log.addHandler(handler)
 
 
+def increment_metrics(namespace):
+    """
+    Increment the metric of any default route that has a metric of 0.
+    :param namespace: The Networking namespace of the container.
+    :return: None. Raises CalledProcessError on error.
+    """
+    with NamedNamespace(namespace) as ns:
```

Comment:
Whilst we aren't trying to address all situations here, I think we could do better about there already being multiple default routes with different metrics.  I think the original design said something like:
-  Check if there is a default with metric 0, and if so increment all default metrics in reverse order.

For example: This gets round there being a default route with metric 0 and metric 1.